presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMinMaxByAggregation.java

```diff
@@ -142,14 +141,8 @@ public void testMaxDoubleDouble()
                 new Signature("max_by", AGGREGATE, parseTypeSignature(StandardTypes.DOUBLE), parseTypeSignature(StandardTypes.DOUBLE), parseTypeSignature(StandardTypes.DOUBLE)));
         assertAggregation(
                 function,
-                null,
-                createDoublesBlock(null, null),
-                createDoublesBlock(null, null));
-
-        assertAggregation(
-                function,
                 2.0,
-                createDoublesBlock(3.0, 2.0, null),
+                createDoublesBlock(3.0, 2.0, 1.0),
```

Comment:
@haozhun and I discussed about this and we thought key should not be nullable. But we can support nullable keys as well I guess. I assume if the key is null we just ignore it?